test/PDPVerifier.t.sol

```diff
@@ -1662,6 +1662,38 @@ contract PDPVerifierE2ETest is Test, ProofBuilderHelper {
         return (pythCallData, price);
     }
 
+    function createPythUnsafeCallData() internal view returns (bytes memory, PythStructs.Price memory) {
+        bytes memory callData = abi.encodeWithSelector(
+            IPyth.getPriceUnsafe.selector,
+            pdpVerifier.FIL_USD_PRICE_FEED_ID()
+        );
+
+        PythStructs.Price memory price = PythStructs.Price({
+            price: 6,
```

Comment:
Doing these fixed flag values isn't the best but its consistent with the current state of the world and gets it done.